crates/fuzzing/src/oracles.rs

```diff
@@ -137,7 +137,21 @@ pub fn instantiate_with_config(
         Err(_) if !known_valid => return,
         Err(e) => panic!("failed to compile module: {:?}", e),
     };
-    let linker = dummy_linker(&mut store, &module);
+    let linker = match dummy_linker(&mut store, &module) {
+        Ok(linker) => linker,
+        Err(e) => {
+            eprintln!("Warning: failed to create host imports: {:?}", e);
+
+            // Currently the only error we want to allow here is ones where we
+            // ran out of resources creating imports. For example memory
+            // creation may not succeed if the host is running low on resources.
+            //
+            // Other errors, however, are bugs in creation of the host resource.
+            let string = e.to_string();
+            assert!(string.contains("Insufficient resources"));
```

Comment:
Would it make sense to have a slightly more specific match here? E.g. "Insufficient resources: memory" or somesuch?